server/config/incident.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  name: {
+  newIncident: {
```

Comment:
not particularly happy with this name - the first page in the workflow is 'New use of force incident' and doesn't nicely cover a unified set of info that we could describe here (which might be a bit of warning in its own right!)

Will consider this a bit further and most likely update in a follow up.